lib/node_modules/@stdlib/stats/incr/wstdev/docs/repl.txt

```diff
@@ -0,0 +1,44 @@
+
+{{alias}}( [mean] )
```

Comment:
Looks like we have documentation inconsistency here. The README does not indicate that a known mean is supported.